contracts/L2Notifier.sol

```diff
@@ -0,0 +1,91 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@umb-network/toolbox/dist/contracts/IRegistry.sol";
+import "@umb-network/toolbox/dist/contracts/IChain.sol";
+
+import "./interface/L2Subscriber.sol";
+
+/// @title Umbrella L2Notifier
+/// @notice Contract that receives a delivery request, verifies data and delivers it
+/// @dev Developers may implement this deliverer by their own.
+contract L2Notifier {
+  IRegistry public immutable registry;
+
+  mapping(bytes32 => uint32) public subscriptions;
+
+  event LogSubscriptionCreated(address indexed subscriber, bytes32 key);
+
+  constructor(address _registry) {
+    require(_registry != address(0x0), "_registry is empty");
+
+    registry = IRegistry(_registry);
+  }
+
+  /// @notice Register new delivery request, to future block
+  /// @param _key bytes32 encoded key
+  /// @param _minBlockId minimum Umbrella's sidechain block number to deliver data
+  function register(bytes32 _key, uint32 _minBlockId) external returns (bool success) {
```

Comment:
No need to return anything here, right? @avoloshko

Reply:
Indeed, we should not return anything